fpsim/locations/calibrate_manual.py

```diff
@@ -311,20 +326,20 @@ def pop_growth_rate(years, population):
 
 
 if do_plot_cpr:
+
         '''
         Plot contraceptive prevalence rate for model vs data
         '''
-
         # Import data
-        data_cpr = pd.read_csv('kenya_cpr.csv') # From UN Data Portal
-        data_cpr = data_cpr[data_cpr['year'] <= 2020] # Restrict years to plot
+        #data_cpr = pd.read_csv(f'{country}_cpr.csv') # From UN Data Portal
```

Comment:
we can probably remove line 334; I think I commented it out when I added this line to the top of the file (with the rest of the data inputs) and forgot to delete it here.

Reply:
@emilykclaps done, thanks